packages/astro-check/README.md

```diff
@@ -0,0 +1,36 @@
+# @astrojs/check ✅
+
+Type checking and diagnostics for Astro projects.
```

Comment:
It might be useful to specify that this powers the `astro check` command, otherwise people might think it's an actual type checker

Reply:
Something like this? I’ll admit I followed the other READMEs that aim to give zero docs, just links.

```suggestion
Provides the `astro check` command for running type checking and diagnostics in Astro projects.
```